### PR TITLE
[NF] ElmerResultsVTU helper class to load results

### PR DESCRIPTION
**Pull request by SebGue**
This PR add a new Elmer class to load the vtu results into pyleecan. 
Since this class has only one method, it could have been done with a function also. But because there are also other convenient Elmer classes, I decided for a class over a function.

### CODE REVIEW

**Comment by ajpina** [comment]
Hi @SebGue ,
What about Cell Data? I see that get_meshsolution() in this PR is loading only Point Data.
Best,
Alejandro

**Comment by SebGue** [comment]
Hello Alejandro,

you are right. Actually this is not implemented. Do you have such data file?
If you upload it, I try to add this feature as soon as possible.

Best regards, Sebastian

**Comment by ajpina** [comment]
Please check out my initial attempt based on your code PR #292 `get_meshsolution` method. It's working so far with my test case but not sure whether is general enough yet.
Best,
Alejandro

File: pyleecan/Methods/Elmer/ElmerResultsVTU/get_meshsolution.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+from ....Classes.MeshSolution import MeshSolution
+from ....Classes.MeshVTK import MeshVTK
+
+from ....Methods.Elmer.ElmerResultsVTU import ElmerResultsVTUError
+
+
+# TODO add groups, see get_meshsolution of MagFEMM
+
+
+def get_meshsolution(self):
```
Comment (Superomeg4) [inline pyleecan/Methods/Elmer/ElmerResultsVTU/get_meshsolution.py:19]: To be more coherent with MagFEMM, I would rename as "build_meshsolution" instead.

**Comment by SebGue** [comment]
@BonneelP I think you can merge this PR now.
@ajpina Sorry for my late reponse. I had a look at your PR. I looks good. As soon as I got some time, I will merge your get_meshsolution to the general ElmerVTU class.

**Comment by BonneelP** [comment]
Thank you for your contribution :)

**Pull request merged**